### PR TITLE
Cargo Hardsuit Market

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

This PR adds several hardsuits to a new "hardsuits" category for the cargo menu. This is part of an ongoing effort to provide reworks to Salvage gameplay. This should pair quite well with merging Frontier's changes to Salvage Expeditions, as well as a variety of other changes I've been talking about. My theory is that if Expeditions are changed to provide money rewards deposited directly to the station's account, then Salvage can benefit the station by A: Providing station funds whether they want to or not. And B: Be encouraged to do so because by getting cargo a shitload of money, they will be able to directly buy the equipment needed to go on higher tier expeditions. And to do this without needing to gamble on wrecks for 2 hours straight, desperately hoping you get the specific wreck that contains that security hardsuit. 

<details><summary><h1>Media</h1></summary>
<p>

image

</p>
</details>

# Changelog

:cl:
- add: Logistics can now directly order a variety of hardsuits. "NT In-Network" hardsuits being substantially cheaper thanks to backroom deals.

### CODE REVIEW

**Comment by VMSolidus** [comment]
I've updated the prices of the hardsuits.

**Review by Peptide90 — APPROVED** [review]

File: Resources/Prototypes/Catalog/Cargo/cargo_hardsuits.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+- type: cargoProduct
+  id: SecurityTsagaanTacsuit
+  icon:
+    sprite: DeltaV/Clothing/OuterClothing/Hardsuits/Combat/medical.rsi
+    state: icon
+  product: CrateSecurityTsagaanTacsuit
+  cost: 5500
+  category: cargoproduct-category-name-hardsuits
+  group: market
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/Cargo/cargo_hardsuits.yml:92]:
```suggestion
  group: market

```

File: Resources/Prototypes/Catalog/Fills/Crates/hardsuits.yml [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# Hardsuits
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/Fills/Crates/hardsuits.yml:1]:
```suggestion
```

File: Resources/Prototypes/Catalog/Fills/Crates/hardsuits.yml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  - type: StorageFill
+    contents:
+    - id: ClothingOuterHardsuitSalvage
+
+# Security
+- type: entity
+  id: CrateSecurityShanlinTacsuit
+  parent: CrateSecgear
+  name: shanlin tacsuit crate
+  description: Contains a single CSA-51a Shanlin tacsuit. Requires Security access to open.
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/Fills/Crates/hardsuits.yml:49]:
```suggestion
  description: Contains a single CSA-51a "Shanlin" tacsuit. Requires Security access to open.
```

File: Resources/Prototypes/Catalog/Fills/Crates/hardsuits.yml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  components:
+  - type: StorageFill
+    contents:
+    - id: ClothingOuterHardsuitSyndieReverseEngineered
+
+- type: entity
+  id: CrateSecurityGuanYuTacsuit
+  parent: CrateSecgear
+  name: guan-yu tacsuit crate
+  description: Contains a single CSA-80UA Guan-Yu tacsuit. Requires Security access to open.
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/Fills/Crates/hardsuits.yml:59]:
```suggestion
  description: Contains a single CSA-80UA "Guan-Yu" tacsuit. Requires Security access to open.
```

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Pull request merged**